.github/workflows/run_examples_js.yml

```diff
@@ -0,0 +1,12 @@
+# Run examples inside JS folders
+name: Run examples JS
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  test-examples:
+    uses: ./.github/workflows/examples_js.yml
+    with:
+      working-directory: ./js
+    secrets: inherit
```

Comment:
Need for this. Any future plan for composable actions with above?

Reply:
I used the same approach as done for python tests, Thought of keep the flow same